userdocs/src/usage/nodegroup-override-announcement.md

```diff
@@ -0,0 +1,29 @@
+# Nodegroup Bootstrap Override For Custom AMIs
+
+This change was announced in the issue [Breaking: overrideBootstrapCommand soon...](https://github.com/weaveworks/eksctl/issues/3563).
+Now, it has come to pass. Please read the attached issue carefully about why we decided to move away from supporting
+custom AMIs without bootstrap scripts or with partial bootstrap scripts.
+
+We still provide a helper! Migrating hopefully is not that painful. `eksctl` still provides a helper script, which when
+sourced, will provide a couple of helpful environment properties and settings. This script is located [here](https://github.com/weaveworks/eksctl/blob/70a289d62e3c82e6177930cf2469c2572c82e104/pkg/nodebootstrap/assets/scripts/bootstrap.helper.sh).
+
+The absolute minimum that needs to be used when overriding though so eksctl doesn't fail is labels! `eksctl`
+relies on a specific set of labels to be on the node, so it can find them. When defining the override, please provide
+this **bare minimum** override command:
+
+```yaml
+    overrideBootstrapCommand: |
+      #!/bin/bash
+
+      source /var/lib/cloud/scripts/eksctl/bootstrap.helper.sh
```

Comment:
do we want to document what will be available from the helper script so they dont have to go read it?

Reply:
Yep, good idea